pkg/clients/builder.go

```diff
@@ -118,8 +123,42 @@ func (b *Builder) BuildOrDie(ctx context.Context) *Clients {
 	b.client.KubeClient = guestKubeClient
 	b.client.KubeInformers = v1helpers.NewKubeInformersForNamespaces(guestKubeClient, b.guestNamespaces...)
 
+	extractApplySpec := func(obj *unstructured.Unstructured, fieldManager string) (*applyoperatorv1.OperatorSpecApplyConfiguration, error) {
```

Comment:
Please take extractApplySpec and extractApplyStatus outside of the function (so they are their own functions) and rename them to extractOperatorSpec and extractOperatorStatus